### PR TITLE
Split for WF setups

### PR DESCRIPTION
**Pull request by vmcj**
There is no clean way to split the group and not share variables.

I encountered this when testing both the online and onprem hosts for the WF where variables can be different such as the network config. For BAPC/NAC/{N,S}NWERC & EUC we might have the same situations

### CODE REVIEW

File: provision-contest/ansible/group_vars/online/all.yml.example [hunk 1]
```diff
@@ -0,0 +1 @@
+../onprem/all.yml.example
```
Comment (nickygerritsen) [inline provision-contest/ansible/group_vars/online/all.yml.example:1]: Does this make sense, if you know copy this symlink, you get another copy of the symlink right? And not the actual file.

Though I'm not sure how to do it better.

Comment (vmcj) [reply provision-contest/ansible/group_vars/online/all.yml.example:1]:
```
vmcj@DESKTOP-QMUMFGG:/tmp$ touch original_file
vmcj@DESKTOP-QMUMFGG:/tmp$ ln -s original_file new_file
vmcj@DESKTOP-QMUMFGG:/tmp$ cp new_file new_2_file
vmcj@DESKTOP-QMUMFGG:/tmp$ ls -atrl | grep _file
-rw-r--r--  1 vmcj vmcj    0 Sep  2 18:07 original_file
lrwxrwxrwx  1 vmcj vmcj   13 Sep  2 18:07 new_file -> original_file
-rw-r--r--  1 vmcj vmcj    0 Sep  2 18:07 new_2_file
```

Not on my system, and my default script would create the needed file already. There is a python script next to it which autofills such example files to the actual `yaml` file.

Comment (nickygerritsen) [reply provision-contest/ansible/group_vars/online/all.yml.example:1]: Ok then I'm mistaken 🙈

Comment (vmcj) [reply provision-contest/ansible/group_vars/online/all.yml.example:1]: @nickygerritsen so this is an approval? I tested and the syntax errors are now gone (so the only question is if we want this...)

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**